### ISSUE TITLE
Home Page Hero Banner

### ISSUE THREAD

**Issue by sarah0arnold** [aemsites/clarkcountynv#1364]
**Component Block:** None

**Design:** https://www.figma.com/design/T2bgro0OiuLqxvm1uLhJZ2/Clark-County-NV?node-id=304-256&t=DFmbW9SuWahfeBhH-0

**User Story:** As a site visitor, I want a visual introduction to the website when I hit the home page so that I can orient myself and find the information I am looking for. 

**Requirements:** 
Home page hero banner is 330px tall below the header navigation element and 148px tall on mobile
No overlay on home page hero
Set max width to 1440 - beyond 1440, there is whitespace on either side of the hero image (design needs to be updated)
Only for home page hero: screen reader should announce alt text on the image

**Acceptance Criteria:** 
1. Given I am viewing the home page on desktop, when the page renders, then the hero banner has a height of 330px below the header. 
2. Given I am viewing the home page on mobile, when the page renders, then the hero banner has a height of 148px below the header.

**Comment by aaronstewart-dev** [comment]
This ticket is dependent on the Header being updated.

**Comment by aaronstewart-dev** [comment]
@sarah0arnold Is there a site-width wrapper for just the hero or for the entire site, including content sections?

**Comment by beccasuchanek** [comment]
@aaronstewart-dev This ticket passed all ACs, see details below:

**Test Page URL:** https://develop--clarkcountynv--aemsites.aem.page/drafts/bsuchanek/tickets/home-page-components

**Status of ACs**
1. ✅ I am viewing the home page on desktop, when the page renders, then the hero banner has a height of 330px below the header. 
<img width="1678" height="552" alt="Image" src="https://github.com/user-attachments/assets/aaf79988-22f2-4eca-8962-0cfb4e6baf5f" />

2. ✅ Given I am viewing the home page on mobile, when the page renders, then the hero banner has a height of 148px below the header.
<img width="1533" height="497" alt="Image" src="https://github.com/user-attachments/assets/37adb634-bc7e-4091-903e-745018a04bee" />

**Comment by sarah0arnold** [comment]
Home Page hero reviewed in UAT session with project team 8/4/25 and passes with no feedback. Closing ticket
